Dev issue 541 slice bar

UnHumbleBen opened the pull request.

dave-doty reviewed (changes requested):
The backbone angles should only be displayed in the side view when the slice bar is on. It's odd to display them otherwise, the slice bar is invisible, and it's not clear which offset the angles are referring to:

![image](https://user-images.githubusercontent.com/19274365/103364303-49cf9480-4a72-11eb-9f2a-2a642650e2b1.png)

dave-doty commented:
Also, please update the README to describe how to use the two new menu items for slice bar and viewing strand details.

UnHumbleBen commented:
> The backbone angles should only be displayed in the side view when the slice bar is on. It's odd to display them otherwise, the slice bar is invisible, and it's not clear which offset the angles are referring to:
> 
> ![image](https://user-images.githubusercontent.com/19274365/103364303-49cf9480-4a72-11eb-9f2a-2a642650e2b1.png)
> Also, please update the README to describe how to use the two new menu items for slice bar and viewing strand details.

Fixed in https://github.com/UC-Davis-molecular-computing/scadnano/pull/564/commits/4b9b4901da0ae31b70bb30c1a41cde6616014aa6.
Also removed backbone edit mode details from the README and mostly reused it for details on the slice bar
